frontend/components/CellInput.js

```diff
@@ -877,6 +884,15 @@ const InputContextMenu = ({ on_delete, cell_id, run_cell, running_disabled, any_
                       ${running_disabled ? html`<span class="enable_cell ctx_icon" />` : html`<span class="disable_cell ctx_icon" />`}
                       ${running_disabled ? html`<b>Enable cell</b>` : html`Disable cell`}
                   </li>
+                  <li
+                      onClick=${toggle_skip_as_script}
+                      title=${skip_as_script
+                          ? "Skip this cell when running the notebook as script, making it exclusive to the notebook"
+                          : "Run this cell also when calling the notebook as a script"}
```

Comment:
I think the descriptions do to match the label and should be switched:

```suggestion
                      title=${skip_as_script
                          : "Run this cell also when calling the notebook as a script"
                          ? "Skip this cell when running the notebook as script, making it exclusive to the notebook"}
                       
```